Sources/AnimalKingdomAPI/AnimalKingdomAPI/Sources/Schema/Schema.swift

```diff
@@ -11,6 +11,12 @@ where Schema == AnimalKingdomAPI.Schema {}
 public protocol InlineFragment: ApolloAPI.SelectionSet & ApolloAPI.InlineFragment
 where Schema == AnimalKingdomAPI.Schema {}
 
+public protocol MutableSelectionSet: ApolloAPI.MutableRootSelectionSet
```

Comment:
If I'm not mistaken I think this PR is missing the changes to `SchemaTemplate.swift` to generate this code. I just rebased my PR (#2313) and this file isn't having these protocols generated. Let's sync on this in the morning.